mlir/include/mlir/Dialect/Rock/utility/builderUtils.h

```diff
@@ -39,6 +39,10 @@ Type getFlattenedType(Type type);
 Value getAsTensor(OpBuilder &builder, Location loc, mlir::Value value,
                   bool isWritable = false);
 
+// Return the type of a boolean vector whose shape is the same as the shape of
+// `v` except that its elements are booleans.
+Type validityVectorShapedLike(Value v);
```

Comment:
maybe a better name : "getBoolVectorOfShape(Value v)" ? 
Point being .. we could be using this elsewhere its not "validity" in future.
(not a strong comment)